.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor
+.idea
```

Comment:
This should really be part of your own global `gitignore` as it's not a file generated by this repo.

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Thanks